src/attribution/onLCP.ts

```diff
@@ -37,7 +37,8 @@ const attributeLCP = (metric: LCPMetric): LCPMetricWithAttribution => {
     const navigationEntry = getNavigationEntry();
     if (navigationEntry) {
       const activationStart = navigationEntry.activationStart || 0;
-      const lcpEntry = metric.entries[metric.entries.length - 1];
+      // The `metric.entries.length` check ensures there will be an entry.
+      const lcpEntry = metric.entries.at(-1)!;
```

Comment:
annoying that this is different behavior and more annoying that complaining about it just exposes the lie of square bracket access not including `undefined` in the type :)

Reply:
Yeah, I was surprised to see that the behavior was different. But on the plus side the warning actually made be double check that there weren't cases where it could be undefined.